lib/diff/lcs/internals.rb

```diff
@@ -44,13 +44,12 @@ def lcs(a, b)
     b_finish = b.size - 1
     vector = []
 
-    # Prune off any common elements at the beginning...
+    # Collect any common elements at the beginning...
```

Comment:
I feel like `collect` is a more proper word, it suggests that we don't discard it but collect it into vector